portfolio/src/main/webapp/index.html

```diff
@@ -25,7 +25,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
 
   </head>
-  <body onload = "getFacts(); getComments();">    
```

Comment:
I couldn't find the definition of getComments() or any other usages of it. Make sure you don't have unused code lying around.

Reply:
Fixed!

Reply:
This was actually the previous version. I removed getComments() in this pull request :D